ObjectContainer/src/Main.java

```diff
@@ -0,0 +1,90 @@
+import ro.sciit.comparators.PersoaneAgeComparator;
+import ro.sciit.comparators.PersoaneNameComparator;
+import ro.sciit.entities.Angajat;
+import ro.sciit.entities.Persoane;
+import ro.sciit.entities.Somer;
+import ro.sciit.entities.Student;
+import ro.sciit.utilities.Adresa;
+import ro.sciit.utilities.Hobby;
+
+import java.util.*;
+
+/**
+ * @author Georgiana
+ * Main class
+ */
+
+public class Main {
+
+    public static void main(String[] args) {
+
+
+        System.out.println("Sorting using age comparator");
+
+        Set<Persoane> compAge = new TreeSet<Persoane>(new PersoaneAgeComparator());
```

Comment:
Well done for using the interface as the variable's type instead of the implementation!